lib/config.py

```diff
@@ -24,6 +24,74 @@
 from lib import utils
 
 LOG = logging.getLogger(__name__)
+BUILD_REPO_ARGS = {
+    '--build-versions-repository-url':
+        dict(help='Build versions repository URL'),
+    '--build-version':
+        dict(help='Select build version from versions repository'),
+    '--build-versions-repo-dir':
+        dict(help='Directory to clone the build versions repository',
+             default='./components'),
+    '--http-proxy':
+        dict(help='HTTP proxy URL'),
+}
+PACKAGE_ARGS = {
+    '--packages -p':
```

Comment:
To make a tuple with a single element, leave a comma at the end: `('--keep-builddir',)`